operator/internal/manifests/internal/alerts/prometheus-alerts.yaml

```diff
@@ -10,21 +10,67 @@ groups:
       runbook_url: "[[ .RunbookURL ]]#Loki-Request-Errors"
     expr: |
       sum(
-          rate(
-              loki_request_duration_seconds_count{status_code=~"5.."}[1m]
-          )
+        rate(
+          loki_request_duration_seconds_count{status_code=~"5.."}[1m]
+        )
       ) by (namespace, job, route)
       /
       sum(
-          rate(
-              loki_request_duration_seconds_count[1m]
-          )
+        rate(
+          loki_request_duration_seconds_count[1m]
+        )
       ) by (namespace, job, route)
       * 100
       > 10
     for: 15m
     labels:
       severity: critical
+  - alert: LokistackWriteRequestErrors
+    annotations:
+      message: |-
+        {{ printf "%.2f" $value }}% of write requests from {{ $labels.job }} are returned with server errors.
+      summary: "At least 10% of write requests to the lokistack-gateway are responded with 5xx server errors."
+      runbook_url: "[[ .RunbookURL ]]#Lokistack-Write-Request-Errors"
+    expr: |
+      sum(
+        rate(
+          http_request_duration_seconds_count{code=~"5..", group="logsv1", handler="push"}[1m]
+        )
+      ) by (namespace, job, tenant)
+      /
+      sum(
+        rate(
+          http_request_duration_seconds_count{group="logsv1", handler="push"}[1m]
```

Comment:
I think I'd rather use `http_requests_total` instead of `http_request_duration_seconds_count`. They _should_ have the same values, but the `http_requests_total` metric is the metric usually used in documentation for these "number of failed requests" counts.